icons.scss

```diff
@@ -710,7 +710,7 @@
 @mixin d2l-icon-tier1-dragger() {
 	@include d2l-icon();
 	&:before {
-		content: url("data:image/svg+xml,%3Csvg%20width%3D%2218%22%20height%3D%2218%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2018%2018%22%3E%0A%20%20%3Cpath%20fill%3D%22%23565a5c%22%20d%3D%22M8%2016v1c0%20.5-.4%201-1%201H6c-.6%200-1-.5-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M13%2016v1c0%20.5-.4%201-1%201h-1c-.6%200-1-.5-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M8%2011v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M13%2011v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M8%206v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1V6c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M13%206v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1V6c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M8%201v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1V1c0-.5.4-1%201-1h1c.6%200%201%20.5%201%201M13%201v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1V1c0-.5.4-1%201-1h1c.6%200%201%20.5%201%201%22%2F%3E%0A%3C%2Fsvg%3E%0A");
+		content: url("data:image/svg+xml,%3Csvg%20width%3D%2218%22%20height%3D%2218%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2018%2018%22%3E%3Cpath%20fill%3D%22%23565a5c%22%20d%3D%22M8%2016v1c0%20.5-.4%201-1%201H6c-.6%200-1-.5-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201m5%200v1c0%20.5-.4%201-1%201h-1c-.6%200-1-.5-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201m-5-5v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201m5%200v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M8%206v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1V6c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201m5%200v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1V6c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M8%201v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1V1c0-.5.4-1%201-1h1c.6%200%201%20.5%201%201m5%200v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1V1c0-.5.4-1%201-1h1c.6%200%201%20.5%201%201%22%2F%3E%3C%2Fsvg%3E");
```

Comment:
Not sure why this also changed...

Reply:
Me neither. Looks like just some (encoded) whitespace was removed